src/components/sidebar/Sidebar/index.tsx

```diff
@@ -14,54 +20,82 @@ import { trackEvent, OVERVIEW_EVENTS } from '@/services/analytics'
 import KeyholeIcon from '@/components/common/icons/KeyholeIcon'
 import OwnedSafes from '../OwnedSafes'
 
-const Sidebar = (): ReactElement => {
+type SidebarProps = {
+  hideSidebar: boolean
+  showCollapseSidebarButton: boolean
+  collapseSideBar: () => void
```

Comment:
Is this a callback? If so it should be called `onCollapse`.

Reply:
its the function that sets in the React state of the sidebar is open or closed:

```
  const [isSidebarCollapsed, setIsSidebarCollapsed] = useState<boolean>(isMobile)

  const collapseSidebar = () => setIsSidebarCollapsed((prev) => !prev)
```

Reply:
Yeah, so it’s a callback.